lib/wings/valkyrie/query_service.rb

```diff
@@ -131,6 +131,19 @@ def find_inverse_references_by(resource: nil, id: nil, property:)
         end
       end
 
+      # Find all parents of a given resource.
+      # @param resource [Valkyrie::Resource] The resource whose parents are being searched
+      #   for.
+      # @return [Array<Valkyrie::Resource>] All resources which are parents of the given
+      #   `resource`. This means the resource's `id` appears in their `member_ids`
+      #   array.
+      def find_parents(resource:)
+        id = resource.alternate_ids.first
```

Comment:
This question lies outside of the scope of this PR, but I would be curious as to why the singular `resource.id` cannot instead be used for the Solr query.